### PR TITLE
Changed tables

### PR DESCRIPTION
**Pull request by YuliiaManivska**

### CODE REVIEW

File: lesson5/app/models/patient.rb [hunk 1]
```diff
@@ -1,4 +1,11 @@
 class Patient < ApplicationRecord
+  validates :first_name, :last_name, :age, presence: true
+  validates :first_name, :last_name, length: { in 2..25 }
+  validates :age, numericality: { only_integer: true,
+    message "%{value} is not a valid" }
```
Comment (olehvavryniv) [inline lesson5/app/models/patient.rb]: I think you forgot `:`. Should be `message: "....`

File: lesson5/app/models/patient.rb [hunk 2]
```diff
@@ -1,4 +1,11 @@
 class Patient < ApplicationRecord
+  validates :first_name, :last_name, :age, presence: true
+  validates :first_name, :last_name, length: { in 2..25 }
+  validates :age, numericality: { only_integer: true,
+    message "%{value} is not a valid" }
+    
   has_many :appointments
   has_many :physicians, through: :appointments
+  has_many :diagnosis_items
+  has_many :physicians, through: :diagnosis_items
```
Comment (olehvavryniv) [inline lesson5/app/models/patient.rb]: You duplicated `has_many :physicians`. It is an error.

File: lesson5/app/models/physician.rb [hunk 3]
```diff
@@ -1,4 +1,12 @@
...
+  validates :first_name, :phone_number, :level, presence: true
+  validates :level, inclusion: { in: %w(junior middle senior),
+    message: "%{value} is not a valid level" }
+  validates :phone_number, format:{ with: /\A\+380\d{9}/,
+    message: "%{value} is not a vailid phone number" }
+    
   has_many :appointments
   has_many :patients, through: :appointments
+  has_many :diagnosis_items
+  has_many :patients, through: :diagnosis_items
```
Comment (olehvavryniv) [inline lesson5/app/models/physician.rb]: Same here. Line duplication.

File: lesson5/db/migrate/20211215130320_add_columns_to_tables.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+class AddColumnsToTables < ActiveRecord::Migration[6.1]
+  def change
+    change_table :patients do |t|
+      t.rename :name, :first_name
+      t.column :last_name, :string
```
Comment (olehvavryniv) [inline lesson5/db/migrate/20211215130320_add_columns_to_tables.rb]: You forgot NOT NULL constraints.

File: lesson5/db/migrate/20211215130320_add_columns_to_tables.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    change_table :physicians do |t|
+      t.rename :name, :first_name
+      t.column :last_name, :string
+      t.column :phone_number, :string
+      t.column :level, :string
+    end
+
+    create_table :diagnosis_items do |t|
+      t.text :content
+      t.timestamps
```
Comment (olehvavryniv) [inline lesson5/db/migrate/20211215130320_add_columns_to_tables.rb]: Where are the references? physician_id and patient_id

**Review by olehvavryniv — CHANGES_REQUESTED** [review]

**Comment by YuliiaManivska** [comment]
@olehvavryniv please, check

**Review by olehvavryniv — APPROVED** [review]

**Pull request merged**